frontend/src/components/admin/menu/SampleEntryConfigMenu.js

```diff
@@ -0,0 +1,297 @@
+import React, { useContext, useState, useEffect, useRef } from "react";
+import {
+  Form,
+  Heading,
+  Toggle,
+  Button,
+  Loading,
+  Grid,
+  Column,
+  Section,
+  Checkbox,
+  DataTable,
+  Table,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableHeader,
+  TableCell,
+  TableSelectRow,
+  TableSelectAll,
+  TableContainer,
+  Pagination,
+} from "@carbon/react";
+import { Save } from "@carbon/icons-react";
+import {
+  getFromOpenElisServer,
+  postToOpenElisServerFullResponse,
+} from "../../utils/Utils";
+import { MenuCheckBox } from "./MenuUtil";
+import { NotificationContext } from "../../layout/Layout";
+import {
+  AlertDialog,
+  NotificationKinds,
+} from "../../common/CustomNotification";
+import config from "../../../config.json";
+import { FormattedMessage, useIntl } from "react-intl";
+import PageBreadCrumb from "../../common/PageBreadCrumb.js";
+
+let breadcrumbs = [{ label: "home.label", link: "/" }];
+function SampleEntryConfigMenu() {
+  const { notificationVisible, setNotificationVisible, addNotification } =
+    useContext(NotificationContext);
+
+  const intl = useIntl();
+
+  const componentMounted = useRef(false);
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(5);
+  const [modifyButton, setModifyButton] = useState(true);
+  const [selectedRowId, setSelectedRowId] = useState(null);
+  const [startingRecNo, setStartingRecNo] = useState(1);
+  const [sampleEntryConfigMenuList, setSampleEntryConfigMenuList] = useState(
+    [],
+  );
+  const [orderEntryConfigurationList, setOrderEntryConfigurationList] =
+    useState([]);
+
+  // async function displayStatus(res) {
+  //   setNotificationVisible(true);
+  //   setIsSubmitting(false);
+  //   if (res.status == "200") {
+  //     addNotification({
+  //       kind: NotificationKinds.success,
+  //       title: intl.formatMessage({ id: "notification.title" }),
+  //       message: intl.formatMessage({ id: "success.add.edited.msg" }),
+  //     });
+  //     var body = await res.json();
+  //     setMenuItem(body);
+  //   } else {
+  //     addNotification({
+  //       kind: NotificationKinds.error,
+  //       title: intl.formatMessage({ id: "notification.title" }),
+  //       message: intl.formatMessage({ id: "error.add.edited.msg" }),
+  //     });
+  //   }
+  // }
+
+  // function handleSubmit(event) {
+  //   event.preventDefault();
+  //   setIsSubmitting(true);
+  //   postToOpenElisServerFullResponse(
+  //     "/rest/menu/menu_patient",
+  //     JSON.stringify(menuItem),
+  //     displayStatus,
+  //   );
+  // }
+
+  function handleModify(event) {
+    event.preventDefault();
+    let sampleEntryConfigEditRedirect =
+      config.serverBaseUrl +
+      `/SampleEntryConfig?ID=${selectedRowId}&startingRecNo=${startingRecNo}`;
```

Comment:
We need also to re-write the SampleEntryConfig config in react. 
You can just keep redirecting to the OLD UI for modifying the values

Reply:
alright mosses :slightly_smiling_face: